### PR TITLE
Adding the core strategy

### PR DESCRIPTION
**Pull request by s-laugh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,37 +1,46 @@
...
 
-```markdown
-Syntax highlighted code block
+Each month we will focus on one core [topic](https://github.com/esdc-devcop/strategy/issues?q=is%3Aopen+is%3Aissue+label%3Atopic) that could be actioned and implemented by a team with less than half a day's work. 
+Each month will build on the previous where it will lead teams to developing applications better and faster though increasing automation and reducing feedback loops.
 
-# Header 1
-## Header 2
-### Header 3
+Throughout all of this wee will need to keep in mind that their are many types of technology being used in applications that will need these implementation;
```
Comment (PhilippeTurcotteGov) [inline README.md]:
```suggestion
Throughout all of this, we will need to keep in mind that their are many types of technology being used in applications that will need these implementations;
```

File: README.md [hunk 2]
```diff
@@ -1,37 +1,46 @@
...
+so our solutions need to be non-tool dependant but offer suggestions and recommendations for tools.
```
Comment (PhilippeTurcotteGov) [inline README.md]:
```suggestion
So our solutions need to be non-tool dependant but offer suggestions and recommendations for tools.
```

File: README.md [hunk 3]
```diff
@@ -1,37 +1,46 @@
...
+- [Management Support](#management-support)
+- [Community Building & Engagement](#community-building-engagement)
 
-**Bold** and _Italic_ and `Code` text
+### Info Sessions
 
-[Link](url) and ![Image](src)
-```
+Holding monthly sessions that will introduce the monthly topic and layout the plan of how to implement the topic.
+These sessions will be hosted similar to 'Ted Talks' or 'Coffee and Learn' with a short Q&A period.
```
Comment (PhilippeTurcotteGov) [inline README.md]:
```suggestion
These sessions will be hosted similar to 'TED Talks' or 'Coffee and Learn' with a short Q&A period.
```

File: README.md [hunk 4]
```diff
@@ -1,37 +1,46 @@
...
+### Management Support
 
-### Support or Contact
+We will be providing support to teams that are keen on implementing these topics but are receiving resistance from management.
+Working with management on what they perceive to be preventing them from accepting the implementation to demonstrate how they can do it now.
 
-Having trouble with Pages? Check out our [documentation](https://help.github.com/categories/github-pages-basics/) or [contact support](https://github.com/contact) and we’ll help you sort it out.
+### Community Building & Engagement
+
+We will keep a focus on keeping the community engaged and involved in the various activities though social media and advertising methods.
```
Comment (PhilippeTurcotteGov) [inline README.md]:
```suggestion
We will also focus on keeping the community engaged and involved in the various activities through social media and advertising methods.
```

**Comment by s-laugh** [comment]
> 
> 
> Looks great! I don't know why, but I love the call to action with the "Vote on Issues". I just requested a few changes for minor stuff like wording or typos.

I took all your suggested edits!

**Pull request merged**